DEPS

```diff
@@ -53,6 +53,9 @@ deps = {
    # build. Eventually, we'll manage these dependencies together with Fuchsia
    # and not have to specific specific hashes.
 
+  'src/apps/mozart':
+   Var('fuchsia_git') + '/mozart' + '@' + '496cc8a179d06b74f37c08382efe139831b0f0a5',
```

Comment:
Each of these dependencies that we pin creates a problem.  If you have this DEPS entry, you can no longer modify mozart.git without a complex dance with the standalone Flutter build.

The path to sanity is to decouple the Fuchsia world and the standalone Flutter world (and to move engine.git into fuchsia.googlesource.com).

> It'll be much more invasive.

I'm not sure why it would be particularly invasive.  We just need to add a few conditions to BUILD.gn files and erase a couple types.  I can take this patch and modify it to do that if you like.